featuretools/computational_backends/feature_tree.py

```diff
@@ -9,6 +9,7 @@
 from featuretools import variable_types
 from featuretools.feature_base import (
     AggregationFeature,
+    GroupByTransformFeature,
```

Comment:
should we add a handler so `GroupByTransformFeature` get split by the `feature.groupby` value?